### ISSUE TITLE
Input data format

### ISSUE THREAD

**Issue by Xuechun-Hu** [Ghailen-Ben-Achour/PointNet2_Segmentation#1]
The code uses S3DIS data loader, does the input need to be converted to .npy first?  The dataset contains 3 files with different densities of pointcloud, and each subfolder corresponds to a single pointcloud and contains the semantic segments of the pointcloud in .ply. Should they all be merged and converted to .npy?

**Comment by Ghailen-Ben-Achour** [comment]
@Xuechun-Hu  You are right. The data should be converted to .npy first. I just added a simple script that converts and normalizes the point coordinates and colors here 
https://github.com/Ghailen-Ben-Achour/PointNet2_Segmentation/blob/main/data_utils/ply_to_np.py
This script merges all .ply points inside the folder (from different classes) and stores them into a single numpy file.
You should do this for every folder.

**Comment by cvhabitat** [comment]
> 



> @Xuechun-Hu You are right. The data should be converted to .npy first. I just added a simple script that converts and normalizes the point coordinates and colors here https://github.com/Ghailen-Ben-Achour/PointNet2_Segmentation/blob/main/data_utils/ply_to_np.py This script merges all .ply points inside the folder (from different classes) and stores them into a single numpy file. You should do this for every folder.

Hi @Ghailen-Ben-Achour,

Thank you for releasing the code. I'm a bit confused about the line below:

`data=np.vstack((data,vertices[:vertices.shape[0]:int(vertices.shape[0]/6476),:]))`

Why 6476 here and what does it mean?

Best wishes
